starknet_py/net/client_models.py

```diff
@@ -182,7 +201,9 @@ class TransactionReceipt:
     # pylint: disable=too-many-instance-attributes
 
     hash: int
-    status: TransactionStatus
+    status: Optional[TransactionStatus]
```

Comment:
I think we might want to start adding comments next to the fields.
A short information `# replaced by execution and finality status in 0.4.0-rc2` (and hence the difference with spec of optional field) might be handy in the future.
wdyt?